src/com/opc_ua/opcua_client_information.cpp

```diff
@@ -22,9 +22,7 @@
 std::string gOpcuaClientConfigFile;
 
 CUA_ClientInformation::CUA_ClientInformation(const std::string &paEndpoint) :
-    mEndpointUrl(paEndpoint), mClient(nullptr), mSubscriptionInfo(nullptr), mMissingAsyncCalls(0), mNeedsReconnection(false), mWaitToInitializeActions(false),
-        mIsClientValid(true),
-        mLastReconnectionTry(0), mLastActionInitializationTry(0), mSomeActionWasInitialized(false) {
+    mEndpointUrl(paEndpoint)
```

Comment:
```suggestion
    mEndpointUrl(paEndpoint) {
```

Syntax Error: `{` required here

Reply:
interesting how I missed this. Apparently I was compiling and testing something different while coding.